proto/public.proto

```diff
@@ -491,11 +491,10 @@ message GatewaysTable {
     string namespace = 1;
     string name = 2;
     string cluster_name = 3;
-    uint64 paired_services = 4;
-    bool alive = 5;
-    uint64 latency_ms_p50 = 6;
-    uint64 latency_ms_p95 = 7;
-    uint64 latency_ms_p99 = 8;
+    bool alive = 4;
```

Comment:
this API breaking change will cause some really nasty stuff to happen when upgrading between edges.  better to leave field tag 4 unused.

Reply:
In light of the recommendation to do all that work in the controller, this api stays the same